ecl/hql/hqlgram.y

```diff
@@ -7341,7 +7342,7 @@ scopeFunctionWithParameters
                             OwnedHqlExpr parms = $4.getExpr();
                             //NB: Do not call createComma() incase the first argument is a dataset
                             if (parms)
-                                $$.setExpr(createValue(no_comma, $1.getExpr(), parms.getClear()), $1);
+                                $$.setExpr(createValue(no_comma, nullptr, $1.getExpr(), parms.getClear()), $1);
```

Comment:
Why is the type $1.getType()?

Reply:
I think the type is actually a nullptr in this case.